CHANGELOG.md

```diff
@@ -10,6 +10,17 @@ Changelog for `ta4j`, roughly following [keepachangelog.com](http://keepachangel
 - Renamed `FixedDecimalIndicator` with `FixedNumIndicator`
 - Moved `BaseBar`, `BaseBarBuilder` and `BaseBarBuilderFactory` to `bars`-package
 - Renamed  `Indicator.getUnstableBars` to  `Indicator.getCountOfUnstableBars`
+- Moved `indicators/AbstractEMAIndicator` to `indicators/averages`-package
+- Moved `indicators/DoubleEMAIndicator` to `indicators/averages`-package
+- Moved `indicators/EMAIndicator` 	to `indicators/averages`-package
```

Comment:
Please remove any extra spaces. While it doesn't show when viewing the Markdown it does when viewed directly.

Reply:
Updated